src/mlpack/tests/init_rules_test.cpp

```diff
@@ -63,20 +60,22 @@ BOOST_AUTO_TEST_CASE(OrthogonalInitTest)
 
   for (size_t i = 0; i < weights.n_rows; ++i)
     for (size_t j = 0; j < weights.n_cols; ++j)
-      BOOST_REQUIRE_SMALL(weights.at(i, j) - orthogonalWeights.at(i, j), 1e-3);
+      REQUIRE((weights.at(i, j) - orthogonalWeights.at(i, j)) == 
+          Approx(0.0).margin(1e-3));
```

Comment:
please wrap this lines with `{ }`

Reply:
Hey I am not sure how you want me to wrap this and i looked for the style guide and couldn't find it. can you write it once and i will take care of the rest.

Reply:
```
  for (size_t i = 0; i < weights.n_rows; ++i)
    for (size_t j = 0; j < weights.n_cols; ++j)
     {
        REQUIRE((weights.at(i, j) - orthogonalWeights.at(i, j)) == 
            Approx(0.0).margin(1e-3));
     }
```

Reply:
totally missed those for loops, got it now.